infrastructure/template.yaml

```diff
@@ -357,7 +357,7 @@ Resources:
   CreateAuthCodeFunction:
     Type: AWS::Serverless::Function
     Properties:
-      CodeUri: ../lambdas/issue-credential
```

Comment:
Should it not have it's own folder? That's the format we've stuck with so far and we should be consistent

Reply:
The lambdas are grouped by state machine so all the lambdas for a state machine are together in a folder. In this case, `../lambdas/match` was the matching state machine